lib/ddtrace/propagation/http_propagator.rb

```diff
@@ -18,6 +18,7 @@ def self.inject!(context, env)
       env[HTTP_HEADER_TRACE_ID] = context.trace_id.to_s
       env[HTTP_HEADER_PARENT_ID] = context.span_id.to_s
       env[HTTP_HEADER_SAMPLING_PRIORITY] = context.sampling_priority.to_s
+      env[HTTP_HEADER_ORIGIN] = context.origin.to_s if context.origin
```

Comment:
Not a big deal, but it's fine if you call `to_s` on `nil`, it will just give you a `''` string, as long as that's acceptable. Although when I look at this, the goal is to not set this header at all if the value is blank.

I might suggest we do the same for sampling priority line above, just as a minor bit of housekeeping and consistency.

Reply:
Yup, using `nil` as a means to say "do not propagate this header"